src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_format.py

```diff
@@ -18,8 +16,8 @@ def build_table_output(result, projection):
     final_list = []
 
     for item in result:
-        def _value_from_path(path):
-            obj = item  # pylint: disable=cell-var-from-loop
+        def _value_from_path(each_item, path):
```

Comment:
@tjprescott please take another look. Assigning value to function from out of the closure is often misleading and error-prone